migtests/scripts/functions.sh

```diff
@@ -54,6 +54,12 @@ run_mysql() {
 	mysql -u ${SOURCE_DB_USER} -p${SOURCE_DB_PASSWORD} -h ${SOURCE_DB_HOST} -P ${SOURCE_DB_PORT} -D ${db_name} -e "${sql}"
 }
 
+run_oracle() {
+	db_name=$1
+	sql=$2
+	echo exit | sqlplus "${SOURCE_DB_USER}/${SOURCE_DB_PASSWORD}@(DESCRIPTION = (ADDRESS = (PROTOCOL = TCP)(HOST = ${SOURCE_DB_HOST}) (PORT = ${SOURCE_DB_PORT}))(CONNECT_DATA = (SID = ${db_name})))" @${sql}
```

Comment:
Factor out the "connection string" into a separate variable named `conn_string` and use it in the cmd:
```
conn_string="${SOURCE_DB_USER}...."
echo exit | sqlplus ${conn_string} @${sql}
```

Reply:
fixed